.github/workflows/mdbook_build.yaml

```diff
@@ -21,12 +21,13 @@ jobs:
         run: |
           curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf -y | sh
           rustup update
+          rustup toolchain install nightly-2025-02-20
           
       - name: Install mdbook and plugins
         run: |
-          cargo binstall --no-confirm --version ${MDBOOK_VERSION} mdbook
-          cargo binstall --no-confirm --version ${MDBOOK_KATEX_VERSION} mdbook-katex
-          cargo binstall --no-confirm --version ${MDBOOK_LINKCHECK_VERSION} mdbook-linkcheck
+          cargo +nightly-2025-02-20 binstall --no-confirm --version ${MDBOOK_VERSION} mdbook
```

Comment:
do we still need these nightly versions here?

Reply:
probably not